vowpalwabbit/fb_parser/src/parse_label.cc

```diff
@@ -2,6 +2,7 @@
 // individual contributors. All rights reserved. Released under a BSD (revised)
 // license as described in the file LICENSE.
 
+#include "vw/core/api_status.h"
```

Comment:
does not seem to be used in this file.

Reply:
never mind

Reply:
It gets used for RETURN_ERROR at line 153/154:
 https://github.com/VowpalWabbit/vowpal_wabbit/pull/4684/files/1895dcca3e9ccd629e3972a024d46d6ce49e141f#diff-0a4181992ca5c75481b37b2f08796914a150ba67c47f01701f66bff87447ec48L153-R154